sources/packages/web/src/components/aest/StudentApplicationOfferingChangeDetails.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <v-card>
+    <v-container>
+      <body-header :title="studentName" class="m-1" />
+      <header-title-value
+        class="mb-2"
+        title="Application#"
+        :value="applicationNumber"
+      ></header-title-value>
+      <header-title-value
+        class="mb-6"
+        title="Location"
+        :value="locationName"
+      ></header-title-value>
+      <content-group-info
+        ><div class="mb-2">
+          <h3 class="category-header-medium primary-color">
+            Reason for change
+          </h3>
+          <p>{{ reasonForChange }}</p>
```

Comment:
I see in Figma `reason`,
![image](https://github.com/bcgov/SIMS/assets/77353155/8cc2ab22-0670-40d2-9c5c-6534518c0b6a) and 
![image](https://github.com/bcgov/SIMS/assets/77353155/1b7c4e68-9e86-4808-9f51-5fb36e18fbad)
Are we following the second one? And in the second one I don't see the `location`

Reply:
Yes, talked to @JasonCTang and the decision is to keep everything as below:
![image](https://github.com/bcgov/SIMS/assets/7859295/bbb0bb21-bad1-4328-a175-aa8b8bb5ac7e)

1) **Reason for change** and **Notes from StudentAid BC** will go one below the other. **Reason for change** will not be on the side.
2) In the card with the Student Name and the Application Number, there will be no Location Name since it is already mentioned at the top along with the Institution Name.